src/components/newsItem.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import { Link } from "gatsby"
+import "./newsItem.scss"
+
+export default function NewsItem({
+  image,
+  date,
+  title,
+  description,
+  id,
+  hoverEffects,
+}) {
+  return (
+    <div className={hoverEffects ? "news-item news-item--hover" : "news-item"}>
+      <div className="news-item__back-panel"></div>
+      <div className="news-item__wrapper">
+        <div className="news-item__image-container">
+          {image && (
+            <div
+              className="news-item__image"
+              style={{
+                backgroundImage: `url("${image}")`,
+              }}
+            ></div>
+          )}
+        </div>
+        {date && <h5>{date}</h5>}
+        {title && <h3>{title}</h3>}
+        <div className="news-item__description-container">
+          {description && <p>{description}</p>}
+          {id && (
+            <Link to={`/news/${id}`}>
```

Comment:
Let's make a note that we will want to create urls from the news article title rather than the id.